src/static/themes/shydactyl/shydactyl.css

```diff
@@ -76,7 +76,7 @@
     display: none;
 }
 
-:root.TridactylThemeShydactyl #completions .BufferCompletionSource table {
+:root.TridactylThemeShydactyl #completions .TabCompletionSource table {
```

Comment:
I don't really see the need for the changes but that's fine. We still have exaliases from `buffer` to `tab` so the mapping makes sense to the user.